internal/init/prompts.go

```diff
@@ -126,9 +133,11 @@ func (p PromptHandler) GetParam(projectParams map[string]string) string {
 			exit.Fatal("Exiting prompt:  %v\n", err)
```

Comment:
Mind adding a bit more info here? Maybe say that it's related to the module properties, and if possible, say which module the problem is with? I'm thinking for `CustomPromptHandler` for example, if the module specifies an invalid string, it's gonna be hard for a user to understand what's going on.